tensorrt_llm/_torch/attention_backend/fmha/flashinfer_trtllm_gen.py

```
@@ -507,6 +507,11 @@ class FlashInferTrtllmGenFmha(PhasedFmha):
         (320, 256),
         (576, 512),
     }
+    # (headDimQk, headDimV, tokens_per_block) whose trtllm-gen MLA decode kernel is
+    # slower than the thop.attention fallback, so this backend declines them.
+    SLOWER_MLA_GENERATION_KERNELS = {
+        (576, 512, 32),
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)